.github/workflows/OCV-PR-4.x-W10.yaml

```diff
@@ -13,14 +13,15 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  EXTRA_CMAKE_OPTIONS: '-DCL_Z_OPTION=/Z7 -DOPENCV_DOWNLOAD_PATH=%BINARIES_CACHE% -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DCMAKE_BUILD_TYPE=Release'
+  EXTRA_CMAKE_OPTIONS: '-DCL_Z_OPTION=/Z7 -DOPENCV_DOWNLOAD_PATH=%BINARIES_CACHE% -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DCMAKE_BUILD_TYPE=Release -DWITH_VULKAN=ON'
```

Comment:
Perhaps we should create a new builder instead of modifying the "default" builder.
Lets keep the "default" build configuration "as is" to ensure that it works.
Wee need something similar to CUDA flow.

Reply:
In my opinion, Vulkan is more like OpenCL. We put the Vulkan header file into `3rdparty` as a third-party library, and then dynamically load Vulkan Lib in runtime.
Therefore, regarding Vulkan CI testing, I think we can do the same as the OpenCL CI test. 

The current version of the Vulkan backend is not good enough, so I set `WITH_VULKAN` to OFF. In the future, we can set `WITH_VULKAN` ON as default, if the VULKAN is faster enough.

Reply:
The final decision from the meeting. Prepare separate CI build configurations (Windows & Mac+MoltenVK).